py3-termcolor.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: py3-termcolor
+  version: 2.3.0
+  epoch: 0
+  description: ANSI color formatting for output in terminal
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - python-3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python-3
+      - py3-setuptools
+      - py3-build
+      - py3-installer
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/termcolor/termcolor
+      tag: ${{package.version}}
+      expected-commit: 0e016d928a1d6eff69ff46a6abdc6058dd412f4c
+
+  - runs: |
+      unset SOURCE_DATE_EPOCH
```

Comment:
Could you add a comment explaining why we need this?

Reply:
makes sense, thanks!